subsys/net/lib/sockets/sockets_inet.c

```diff
@@ -2584,6 +2653,23 @@ int zsock_setsockopt_ctx(struct net_context *ctx, int level, int optname,
 
 			break;
 
+		case IP_RECVTTL:
+			if (IS_ENABLED(CONFIG_NET_IPV4) &&
+				IS_ENABLED(NET_CONTEXT_RECV_HOPLIMIT)) {
```

Comment:
Please align the `IS_ENABLED` macros (could be also that GH renders this incorrectly)